includes/polling/mempools/fortios.inc.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+// FortiOS Mempool
+echo 'FortiOS Mempool';
+
+$mempool['total'] = (snmp_get($device, 'FORTINET-FORTIMANAGER-FORTIANALYZER-MIB::fmSysMemCapacity.0', '-OvQ') * 1024);
```

Comment:
Please convert these two snmp_get calls to snmp_get_multi_oid(). You need to specify the MIB in the correct arg of that new function rather than tagging it onto the OID.